modal/mount.py

```diff
@@ -810,3 +810,110 @@ def _is_modal_path(remote_path: PurePosixPath):
         if is_modal_path:
             return True
     return False
+
```

Comment:
Tried moving this and the client package mounts into a separate module, but ended up in import loop hell and backed away from it for now.

Reply:
Yeah I think I've run into similar issues before! Since we no longer have any public API defined in this module I think it's tolerable to have more of this internal stuff.